### DISCUSSION TITLE
Missing TSS

### DISCUSSION THREAD

**Discussion by drxtreem** [SauceLLC/sauce4strava#180] [General]
In the Sauce Performance under Fitness Tracking I see in the Training box on the left it shows "Missing TSS: 520 activities". In looking at those activities on Strava, they all have the TSS amount listed. How can I re-sync to fetch the complete data? My "Fitness Tracking" is only going back 2.5 years even though my Strava has almost 10 years of data.
![Stava Sauce image](https://github.com/SauceLLC/sauce4strava/assets/154762783/3f6b29ab-bac8-49c1-92cf-ad6b67b9c9f5)

**Comment by mayfield** [comment]
Hi @drxtreem is your account still in the sync process?  This can take a very long time the first time because of Strava's rate limits (literally days).  The sync panel  or the lower left of that page should have more details about the current sync status.

**Reply by drxtreem** [reply]
Thanks for helping. The sync status neck to the athlete's name has the arrows turning in the circle. However, the sync status in the bottom left of the page (activity sync) does not have the arrows turning in the circle. Shows processed 694 of 2,234 activities. It hasn't synced any more in the past two days.

**Reply by mayfield** [reply]
694 is right under the 700 limit for a 24 hour period.  So I'm guessing this is your first sync and it's waiting for Strava to allow more network activity.  If you click the spinning sync icon on the top left it should show the status with something about when it will resume or being suspended.  This is all normal.  The first sync is very slow because of Strava rate limits.

**Reply by drxtreem** [reply]
Ahhh. I see. Thanks much.
Just one other question... Under "Training" section, it shows that I am "Missing TSS:" 909 activities. Not sure why since all my activities in Strava have a TSS score. Any ideas why that would be?

**Reply by mayfield** [reply]
Once the update is complete it should take a full pass to update the TSS.  There might be a few missing TSS because of lacking power data, etc. But before you tackle that I'd let the sync finish.

**Comment by drxtreem** [comment]
Thanks. Appreciate the help.
